### PR TITLE
Minutes for 2019-10-11

### PR DESCRIPTION
**Pull request by Adimote**

### CODE REVIEW

File: SR2020/2019-10-11.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Meeting 2019-10-11 Notes
+- Speak to Southampton SRO ask if we can store Kit until kickstart.
+- *Action Point*: AndyBS Define a process for kickstart:
+  - Update the disclaimer form to have box part codes.
+- Question on who actually handles the kit distribution
```
Comment (RealOrangeOne) [inline SR2020/2019-10-11.md]: If this is a question, we should get it answered as soon as possible!

File: SR2020/2019-10-11.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  - It feels weird to hand over to the Competition team then hand support straight over to the kit team.
+- Budget is reviewable now
+- * Action Point*: andybs: Remove irrelevant people from the budget channel.
```
Comment (PeterJCLaw) [inline SR2020/2019-10-11.md]: Github doesn't enjoy rendering this as it's currently formatted.
```suggestion
  - *Action Point*: andybs: Remove irrelevant people from the budget channel.
```

File: SR2020/2019-10-11.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+- Arena
+  - SR arena as 8x8 is too big for the cube with a 1m space around the edge for screens.
+  - We could use the SUSU truss?
+    - Smaller arenas might be possible just with SUSU truss.
+  - We’ll delegated this problem to Alistair.
+- Tshirts
+  - AndyBs to find & design t-shirts. We need them ordered ASAP so they can be shipped with kits.
+- We need a meeting with the trustees (i.e. to discuss payments, volunteer recruiting, etc..)
+  - *Action Point* Andy Busse to sort out a meeting.
+
```
Comment (PeterJCLaw) [inline SR2020/2019-10-11.md]: We also discussed the pending teams kickstart locations email https://github.com/srobo/team-emails/pull/29.

Comment (PeterJCLaw) [reply SR2020/2019-10-11.md]: Ah, just spotted that this was alluded to above, maybe clarify?

**Pull request merged**